tools/timemachine/internal/cmd/get.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This source code is provided 'as is' and no warranties are given as to title or non-infringement, merchantability
+// or fitness for purpose and, to the extent permitted by law, all liability for your use of the code is disclaimed.
+// This source code is governed by Apache License 2.0 that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"context"
+
+	"github.com/iotexproject/go-pkgs/byteutil"
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/blockchain/block"
+	"github.com/iotexproject/iotex-core/blockchain/filedao"
+	"github.com/iotexproject/iotex-core/db"
+	"github.com/iotexproject/iotex-core/pkg/log"
+	"github.com/iotexproject/iotex-core/state/factory"
+	"github.com/iotexproject/iotex-core/tools/timemachine/miniserver"
+)
+
+// get represents the get command
+var get = &cobra.Command{
+	Use:   "get",
+	Short: "Show current height of trie.db and chain.db",
+	Args:  cobra.NoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+
+		cfg := miniserver.Config()
```

Comment:
config file path should be from command parameter, same for genesis file path

Reply:
fixed